sys/net80211/ieee80211_scan_sw.c

```diff
@@ -53,7 +53,7 @@ __FBSDID("$FreeBSD$");
 #include <net/bpf.h>
 
 struct scan_state {
-	struct ieee80211_scan_state base;	/* public state */
+	struct ieee80211_scan_state base __subobject_use_container_bounds; /* public state */
```

Comment:
For cases like this where you have the first member of a derived class being an instance of a base class I prefer to use `__subobject_member_used_for_c_inheritance`.  I haven't looked at all of the SO annotations in detail, but this one at least looks to likely be a C inheritance case.

Reply:
Sounds good. This seems to be inheritance case.